Umbraco-Cloud/Databases/index.md

```diff
@@ -3,6 +3,28 @@ When working with Umbraco Cloud, the way you work with databases might differ fr
 
 So when you clone a site locally, Umbraco Cloud automatically creates a local database and populates it with data from your website running on the Cloud. If you don't specify anything before starting up your site locally, it'll be a SQL CE database that lives in the /App_Data folder. If you wish to use a local SQL Server instead, you can update the connection string in the web.config, but it's important that you do so before your site start up the first time.
 
+## LocalDB
+By default when Umbraco Cloud restores a local database it will be an Umbraco.sdf file in `/App_Data` folder. However if LocalDB is installed and configurated, the restore will create a Umbraco.mdf file. To use LocalDB ensure `applicationHost.config` is configurated with `loadUserProfile="true"` and `setProfileEnvironment="true"`: https://blogs.msdn.microsoft.com/sqlexpress/2011/12/08/using-localdb-with-full-iis-part-1-user-profile/
+
+```
+<add name="ASP.NET v4.0" autoStart="true" managedRuntimeVersion="v4.0" managedPipelineMode="Integrated">
+   <processModel identityType="ApplicationPoolIdentity" loadUserProfile="true" setProfileEnvironment="true" />
+</add>
+```
+
+Usually `applicationHost.config` is located in this folder for IIS:
+`C:\Windows\System32\inetsrv\config`
+
+and in one of thes folders for IIS Express:
+
+`C:\Users\<user>\Documents\IISExpress\config\`
+
+If you're using Visual Studio 2015+ check this path:
+`$(solutionDir)\.vs\config\applicationhost.config`
+
+In Visual Studio 2015+ you can also configure which applicationhost.config file is used by altering the `<UseGlobalApplicationHostFile>true|false</UseGlobalApplicationHostFile>` setting in the project file (eg: MyProject.csproj). (source: MSDN forum)
+```
```

Comment:
Hi Bjarne,
This is excellent!!
Looks like these (```) adds the rest of the article to a code block.
Could you take them out? Then I'll have this merged in.

-Sofie

Reply:
It should be fixed now :)
https://github.com/bjarnef/UmbracoDocs/blob/6ef06a9b83a6321d7de1ab09f6a913d77c5daefe/Umbraco-Cloud/Databases/index.md

Reply:
Thx 😁